app/services/hyrax/autopopulation/record_persistence.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Autopopulation
+    class RecordPersistence
+      attr_accessor :data, :account
+
+      # A wrapper that allows saving to Redis when using Hyrax & to  Activerecord for Hyku
+      # data is hash that has
+      # 3 keys doi_list and orcid_list are arrays & hyrax_orcid_settings is a hash
+      #
+      #   Example
+      #
+      # { "settings" => {
+      #      "orcid_list" => ["0000-0002-0787-9826A"],
+      #       "doi_list" => ["10.36001/ijphm.2018.v9i1.2693"],
+      #        "hyrax_orcid_settings" => { }
+      #  }
+      #  }
+      #
+      def save(data, account = nil)
+        @data = data
+        @account = account
```

Comment:
Do you need these as instance variables? They are being passed to their respective methods as arguments.

Reply:
They can go